db/seeds.rb

```diff
@@ -87,7 +87,7 @@ def load_test_users
       suburb_file.pos = 0 if suburb_file.eof?
       row = CSV.parse(suburb_file.readline)
 
-      suburb, country, state, latitude, longitude = row[0]
+      suburb, _country, _state, latitude, longitude = row[0]
```

Comment:
They indicate that the variable is not going to be used. That's it's okay for linter to ignore this assignment to a useless variable.